client/src/components/graph/graph.js

```diff
@@ -757,7 +767,35 @@ class Graph extends React.PureComponent {
             right: globals.leftSidebarWidth
           }}
         >
-          <div id="graphAttachPoint" />
+          <div id="graphAttachPoint">
+            <GraphOverlayLayer
+              cameraTF={cameraTF}
+              modelTF={modelTF}
+              projectionTF={projectionTF}
+              graphPaddingRightLeft={this.graphPaddingRightLeft}
+              graphPaddingTop={this.graphPaddingTop}
+              responsive={responsive}
+            >
+              <CentroidLabels
+                labels={centroidLabels.labels}
+                dilatedValue={pointDilation.categoryField}
+                colorAccessor={colorAccessor}
+                dispatch={dispatch}
+              />
+            </GraphOverlayLayer>
+
+            <svg
```

Comment:
What advantage do we get by moving this into the graph?

Reply:
Prior to this, the lasso lived on the same SVG element inside of `<div id="graphAttachPoint">`. By creating the new SVG layer for lasso, it allowed for both the lasso, the camera, and the labels to grab pointer events when needed

Reply:
makes sense.